### ISSUE TITLE
Concatenation on multiple lines

### ISSUE THREAD

**Issue by pmishev** [djoos/Symfony-coding-standard#44]
It seems impossible now to do a string concatenation on multiple lines, without getting a warning. 
For example like this:

```
$str = 'some long text that fills the whole line'
          .'some more text';
```

I propose fixing this in the ruleset:

```
    <rule ref="Squiz.Strings.ConcatenationSpacing">
        <properties>
            <property name="ignoreNewlines" value="true"/>
        </properties>
    </rule>
```

**Comment by razbakov** [comment]
You could do it something like this:

```
$str  = 'some long text that fills the whole line';
$str .= 'some more text';
```

**Comment by COil** [comment]
Had the same error, wondering what is the best. Is this rule stated somewhere?

**Comment by vdoria** [comment]
I am facing the same problem. I fail to see where the coding standards say, that it doesn't allow any spaces around concatenation at all???? Also using the variable name over and over makes the code less readable

`Add a single space around binary operators (==, &&, ...), with the exception of the concatenation (.) operator;`

The exception does imho not say its not allowed but that single space doesn't apply to concatenations (in other words the absence of the rule here means you can use as many as you want... !)

**Comment by Soullivaneuh** [comment]
IMHO, this rule should not throw any error if on new lines.

Or maybe add an option for this?

**Comment by smcjones** [comment]
This is a misreading of the coding standards.

> Add a single space around binary operators (==, &&, ...), with the exception of the concatenation (.) operator;

That doesn't sound like it means "do not allow even a single space for concatenation operators".
